libraries/ms-common/test/test_can_hw.c

```diff
@@ -12,6 +15,7 @@ static volatile size_t s_rx_len;
 
 static void prv_handle_rx(void *context) {
   while (can_hw_receive(&s_rx_id, &s_extended, &s_rx_data, &s_rx_len)) {
+    LOG_DEBUG("RX msg 0x%" PRIu32 " (extended %d) %zu bytes\n", s_rx_id, s_extended, s_rx_len);
```

Comment:
Are we keeping this or was this just for your own debugging. I don't mind either way?

Reply:
I think it's useful information. I did bother to use inttypes. :)